openmetadata-ui/src/main/resources/ui/src/pages/PoliciesPage/PoliciesDetailPage/PoliciesDetailPage.tsx

```diff
@@ -0,0 +1,172 @@
+/*
+ *  Copyright 2021 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Card, Col, Empty, Row, Table, Tabs } from 'antd';
+import { ColumnsType } from 'antd/lib/table';
+import { AxiosError } from 'axios';
+import { isEmpty, uniqueId } from 'lodash';
+import React, { useEffect, useMemo, useState } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { getPolicyByName } from '../../../axiosAPIs/rolesAPIV1';
+import Description from '../../../components/common/description/Description';
+import RichTextEditorPreviewer from '../../../components/common/rich-text-editor/RichTextEditorPreviewer';
+import TitleBreadcrumb from '../../../components/common/title-breadcrumb/title-breadcrumb.component';
+import Loader from '../../../components/Loader/Loader';
+import {
+  GlobalSettingOptions,
+  GlobalSettingsMenuCategory,
+} from '../../../constants/globalSettings.constants';
+import { Policy } from '../../../generated/entity/policies/policy';
+import { EntityReference } from '../../../generated/type/entityReference';
+import { getEntityName } from '../../../utils/CommonUtils';
+import { getRoleWithFqnPath, getSettingPath } from '../../../utils/RouterUtils';
+import { showErrorToast } from '../../../utils/ToastUtils';
+import './PoliciesDetail.less';
+
+const { TabPane } = Tabs;
+
+const PoliciesDetailPage = () => {
+  const { fqn } = useParams<{ fqn: string }>();
+
+  const [policy, setPolicy] = useState<Policy>({} as Policy);
+  const [isLoading, setLoading] = useState<boolean>(false);
+
+  const breadcrumb = useMemo(
+    () => [
+      {
+        name: 'Policies',
+        url: getSettingPath(
+          GlobalSettingsMenuCategory.ACCESS,
+          GlobalSettingOptions.POLICIES
+        ),
+      },
+      {
+        name: fqn,
+        url: '',
+      },
+    ],
+    [fqn]
+  );
+
+  const fetchPolicy = async () => {
+    setLoading(true);
+    try {
+      const data = await getPolicyByName(fqn, 'owner,location,teams,roles');
+      setPolicy(data ?? ({} as Policy));
```

Comment:
can you handle the missing data scenario?
```suggestion
if(data){
      setPolicy(data ?? ({} as Policy));
      }else {
      throw jsonData[api-error]
      }catch (error) {
      setPolicy({} as Policy);
      showErrorToast(error as AxiosError);
      }
```

Reply:
@ShaileshParmar11 , if API is successful then there should be some response, so we don't need to add extra checks,

Reply:
make sense thanks!